README.md

```diff
@@ -296,9 +296,12 @@ In the example above, the compiler would search for a module interface file firs
 
 #### `--base-library=DIR`, `-b DIR`
 
-Predefined data types and operations are not build directly into the compiler but are part of the *base library* that accompanies the compiler.
-The compiler uses the same mechanism that is used to load module dependencies to load modules from the base library, i.e., module interface files.
-In contrast to automatically generated module interface files, the base library does not use the JSON file format but TOML since the module interfaces of the base library are maintained manually and TOML is a more user friendly format.
+Predefined data types and operations are not built directly into the compiler but are part of the *base library* that accompanies the compiler.
+
+To load module dependencies from the base library, i.e. module interface files, the compiler uses the same mechanism that is used to load module dependencies.
+
+In contrast to automatically generated module interface files, the base library does not use the JSON file format. Since the module interfaces of the base library are maintained manually, the more user-friendly format TOML is used.
+
 The module interface file format is documented in [`doc/ModuleInterfaceFileFormat.md`][doc/ModuleInterfaceFileFormat.md].
```

Comment:
The extra paragraphs don't really improve the readability of the text in this case. It's okay to move the first sentence into its own paragraph but since all other sentences discuss the module interface file format for the base library, they should be in the same paragraph.

Reply:
I didn't actually mean to add extra paragraphs, just restructure the sentences. I'll remove the extra paragraphs.